src/tests/wscript

```diff
@@ -1,41 +1,218 @@
 #!/usr/bin/env python
 # encoding: utf-8
 
+import subprocess
+import os
+
+all_combinations_init_code = """#include <boost/fusion/container/vector.hpp>
```

Comment:
This template should be in its own file that python can load and fill (it would be much easier to edit).